File: DASTRUCT/dupeFilter.c
#include <stdbool.h>
#include <stdio.h>

int main(int argc, char const *argv[]){
    int nums[10];
    bool duplicate;

    printf("Enter 10 numbers:\n");
    for (int i = 0; i < 10; ++i) {
        duplicate = false;

        scanf("%d", nums + i);
        
        for (int j = 0; j < i; ++j) {
            if (nums[j] == nums[i]) {
                duplicate = true;
                break;
            }
        }
        if (!duplicate) {
            printf("%d ", nums[i]);
        }
    }
    printf("\n");

    return 0;
}

File: DASTRUCT/dupeFilter.py
print("Enter 10 numbers:")

nums = input().split()

for i in range(len(nums)):
    duplicate = False # 2, 4, 2 ,6

    for j in range(i):
        if nums[j] == nums[i]:
            duplicate = True
            break

    if not duplicate:
        print(nums[i], end=" ")
print()